website/content/docs/sync/azurekv.mdx

```diff
@@ -25,16 +25,20 @@ which gives sufficient access to manage secrets. For more information, see the [
 When Azure credentials for the service principal have been obtained and it has access to Azure Key Vault,
 Vault can be configured to create an Azure destination, then begin to associate your secrets with this destination.
 
-1.	Configure the sync destination:
+1.	Configure the sync destination.
+
```

Comment:
Is it preferred to use periods at the end of step descriptions instead of colons?